business_objects/record.py

```diff
@@ -571,3 +571,14 @@ def __get_order_by(project_id: str, first_x: int = 3) -> str:
     if order != "":
         order = "ORDER BY " + order
     return order
+
+
+def get_first_no_text_column(project_id: str, xfkey: str) -> str:
```

Comment:
name of the function is imo misleading (I first thought this only selects the no text column)

however, it selects the record data and the attribute name of the first non-text attribute.